UpsertedProfileOrchestrator/handler.ts

```diff
@@ -178,6 +181,23 @@ export const getUpsertedProfileOrchestratorHandler = (params: {
           } as SendWelcomeMessageActivityInput
         );
       }
+      // Create messages on specific queues when a user profile become enabled
+      for (const serviceQueueName of params.notifyOn) {
+        try {
+          yield context.df.callActivityWithRetry(
```

Comment:
a bit of optimization: we may want to dispatch them in parallel using `context.df.Task.all`. Also, should we fire&forget them? I think we don't want the job to fail because one of these fails

Reply:
Fixed in 7e120b7. We are handling the Activity execution errors into the try-catch block. If an error occurs we only log the errors and the orchestrator execution continue.

Reply:
`notifyOn` is now a optional `NonEmptyArray` because `context.df.Task.all` rise an exception when an empty array of task is provided. Fixed in 5b9cfeb